spec/reel/session_spec.rb

```diff
@@ -135,17 +146,11 @@
   end
 
   it "encryption/decryption are performing well" do
-    orig_value = "Original"
-    cipher = OpenSSL::Cipher::AES128.new :CBC
-    cipher.encrypt
-    cipher.key = key
-    cipher.iv = iv
-    encrypt = URI.encode_www_form_component Base64.encode64 (cipher.update(orig_value) + cipher.final)
-    cipher.decrypt
-    cipher.key = key
-    cipher.iv = iv
-    encrypt = Base64.decode64 URI.decode_www_form_component encrypt
-    decrypt = cipher.update(encrypt) + cipher.final
-    expect(decrypt).to eq orig_value
+    original_value = "test"
+    c = crypto.new
+    expect(c.decrypt c.encrypt original_value).to eq original_value
+    encrypt_val = c.encrypt original_value
+    c.config[:secret_key] = "change"
+    expect(c.decrypt encrypt_val).to_not eq original_value
```

Comment:
Test failed at line 154 : cause => while setting key,iv by default their length is >16 so, I think they may be truncating and changing key is not affection.
Correcting this in next commit.
/cc: @kenichi @digitalextremist